perfscale_regerssion_ci/scripts/custom_workload_env.sh

```diff
@@ -0,0 +1,8 @@
+# Where to clone e2e-benchmarking from
+export E2E_BENCHMARKING_REPOSITORY=${E2E_BENCHMARKING_REPOSITORY:-"https://github.com/cloud-bulldozer/e2e-benchmarking"}
+export E2E_BENCHMARKING_BRANCH=${E2E_BENCHMARKING_BRANCH:-"master"}
+# Common ENVs of custom workload for kube-burner
+export WORKLOAD=${WORKLOAD:-"custom"}
+export INDEXING=${INDEXING:-"false"}
+export METRICS_PROFILE=${METRICS_PROFILE:-""}
+export ALERTS_PROFILE=${ALERTS_PROFILE:-""}
```

Comment:
Should we add in the standard metrics profile under metrics-profiles? 

Might be also good to add in the defaults that are set up in our jenkins kube-burner job 
export COMPARISON_CONFIG=clusterVersion.json podLatency.json podCPU-avg.json podCPU-max.json podMemory-avg.json podMemory-max.json
export GSHEET_KEY_LOCATION=${GSHEET_KEY_LOCATION} (not sure how we can easily set this) 
export EMAIL_ID_FOR_RESULTS_SHEET=${EMAIL_ID_FOR_RESULTS_SHEET} (need to figure out how to easily set this) 
export GEN_CSV=true

Reply:
Added 'metrics-profiles/metrics.yaml' as the default metrics profile.
For the rest, I think we can handle them in the Jenkins job like kube-burner Jenkins job does.